test/keyboard.spec.ts

```diff
@@ -127,7 +127,7 @@ describe('Keyboard', function () {
       await page.evaluate(() => document.querySelector('textarea').value)
     ).toBe('嗨a');
   });
-  it('should report shiftKey', async () => {
+  itFailsFirefox('should report shiftKey', async () => {
```

Comment:
Is there a way to only mark it failing on MacOS?

Reply:
Not built-in, no — we haven’t needed it until now. You could add another `itFailsFirefoxOnMacOS` helper based on `process.platform === 'darwin'`

Reply:
I'm ok with leaving it as is for now. Note that it is also marked as `pass` or `fail` on our side. We can re-enable when the underlying problem is fixed.